customer_pricing_in_product_view/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Customer Pricing in Product View
+================================
+
+This module was written to extend the functionality of products and partners to add a
+'Product Pricing' button to the partner view which when clicked will open the product list
+with the pricelist set to the customers pricelist.
+
+Installation
+============
+
+To install this module, you need to:
+
+* do nothing special
+
```

Comment:
@gurneyalex I wrote it as a joke initially, however the more I think about it I think it should stay in or we should have some standard wording of the same effect and add it to quality tools.  Telling a user what they don't need to do is equally valuable as telling them what they do need to do, and if we put it everywhere and make consistent it will prompt module developers when it changes.

Maybe a simple line like, This Module has no specific installation requirements, and for the one below, This module requires no post installation configuration